<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>restraints_form</class>
 <widget class="QWidget" name="restraints_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>510</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>341</width>
     <height>411</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="2">
     <widget class="QCheckBox" name="checkBox_4">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QCheckBox" name="checkBox_3">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_2"/>
    </item>
    <item row="1" column="2">
     <widget class="QCheckBox" name="checkBox_2">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="lineEdit_6"/>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="lineEdit_5"/>
    </item>
    <item row="0" column="2">
     <widget class="QCheckBox" name="checkBox">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QCheckBox" name="checkBox_5">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEdit_4"/>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_3"/>
    </item>
    <item row="5" column="2">
     <widget class="QCheckBox" name="checkBox_6">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>     X</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>     Y</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>     Z</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Moment X</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Moment Y</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Moment Z</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
